### PR TITLE
feat: Sortable issues table, improved issues table heading

### PR DESCRIPTION
**Pull request by stalgiag**
see #1251 
see #1252 

This PR makes the issues table sortable and filterable by issue status. It also updates the heading to reveal the count of open/closed issues.

### CODE REVIEW

File: client/components/SortableIssuesTable/index.jsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+  };
+
+  const renderTableHeader = () => (
+    <thead>
+      <tr>
+        {[
+          { field: SORT_FIELDS.AUTHOR, title: 'Author' },
+          { field: SORT_FIELDS.TITLE, title: 'Issue' },
+          { field: SORT_FIELDS.STATUS, title: 'Status' },
+          { field: SORT_FIELDS.AT, title: 'AT' },
```
Comment (howard-e) [inline client/components/SortableIssuesTable/index.jsx:110]:
```suggestion
          { field: SORT_FIELDS.AT, title: 'Assistive Technology' },
```

**Review by howard-e — APPROVED** [review]
Looks good! Left a small suggestion. Should be easy to resolve #1250 afterwards with this, with how well contained it is!

**Comment by howard-e** [comment]
@stalgiag looks like a recent w3c/aria-at test change got merged. Noticed the same on `development` so snapshots need an update. Feel free to merge afterwards

**Comment by stalgiag** [comment]
> @stalgiag looks like a recent w3c/aria-at test change got merged. Noticed the same on `development` so snapshots need an update. Feel free to merge afterwards

Yep I am updating now. Thank you!

**Pull request merged**